backend/protocol_rpc/health.py

```
@@ -0,0 +1,70 @@
+    # Check database connectivity if possible
+    db_status = "unknown"
+    try:
+        from flask import current_app
+        from sqlalchemy import text
+        if hasattr(current_app, 'extensions') and 'sqlalchemy' in current_app.extensions:
+            db = current_app.extensions['sqlalchemy']
+            with db.engine.connect() as conn:
+                conn.execute(text("SELECT 1"))
+                conn.commit()
+            db_status = "healthy"
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)